components/core/src/crypto/keys/sym_key.rs

```diff
@@ -28,6 +29,12 @@ use super::super::{SECRET_SYM_KEY_SUFFIX, SECRET_SYM_KEY_VERSION, hash};
 
 pub type SymKey = KeyPair<(), SymSecretKey>;
 
+impl fmt::Debug for SymKey {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "SymKey")
```

Comment:
Should we include some sort of identifier here to differentiate in debug output which SymKey is being displayed?